### PR TITLE
[Enhancement] metric cpu_use_ratio uses growth value

### LINKED ISSUE #13734: metric value of `starrocks_be_resource_group_cpu_use_ratio` is not instantaneous value
### Steps to reproduce the behavior (Required)
now in `be_ip:http_port/metrics` display the last value of `starrocks_be_resource_group_cpu_use_ratio` insteand of the instantaneous value as starrocks doc.

### Expected behavior (Required)
display instantaneous value of resource group CPU usage ratio
### Real behavior (Required)
display the last value of resource group CPU usage ratio
### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
2.3.3

### PR DESCRIPTION
**Pull request by ZiheLiu**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #13734.

## Problem Summary(Required) ：

The metric `cpu_use_ratio` of workgroup shows the cpu usage proportion for the **accumulated** cpu runtime, but it is expected to show the proportion for an interval.

Therefore, modify it to calculate the proportion for the interval between two calls to `update_metric`.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 2.5
  - [x] 2.4
  - [x] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by ZiheLiu** [comment]
run starrocks_admit_test

**Comment by ZiheLiu** [comment]
@Mergify backport branch-2.3 branch-2.4 branch-2.5

**Review by satanson — APPROVED** [review]

File: be/src/exec/workgroup/work_group.cpp [hunk 1]
```diff
@@ -25,6 +25,17 @@ bool WorkGroupSchedEntity<Q>::is_sq_wg() const {
     return _workgroup->is_sq_wg();
 }
 
+template <typename Q>
+void WorkGroupSchedEntity<Q>::incr_runtime_ns(int64_t runtime_ns) {
+    _vruntime_ns += runtime_ns / cpu_limit();
```
Comment (mofeiatwork) [inline be/src/exec/workgroup/work_group.cpp:30]: Can the `cpu_limit` be zero ?

**Review by murphyatwork — APPROVED** [review]

File: be/src/exec/workgroup/work_group.cpp (hunk 1 above)
Comment (ZiheLiu) [reply be/src/exec/workgroup/work_group.cpp:30]: No, FE guarantees that `cpu_limit` is positive at analyzing.
```java
    public static void analyzeProperties(ResourceGroup resourceGroup, Map<String, String> properties) throws SemanticException {
        for (Map.Entry<String, String> e : properties.entrySet()) {
            String key = e.getKey();
            String value = e.getValue();
            if (key.equalsIgnoreCase(ResourceGroup.CPU_CORE_LIMIT)) {
                int cpuCoreLimit = Integer.parseInt(value);
                int avgCoreNum = BackendCoreStat.getAvgNumOfHardwareCoresOfBe();
                if (cpuCoreLimit <= 0 || cpuCoreLimit > avgCoreNum) {
                    throw new SemanticException(String.format("cpu_core_limit should range from 1 to %d", avgCoreNum));
                }
       }
```

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**